BardiParsi/GikUpFrontendMobile
# Install expo

### Description
This PR sets up Expo for the React Native portion of the project.

### Changes
- Installed Expo to manage the React Native project
- Updated project structure to align with Expo's configuration and conventions
- Modified `package.json` scripts to work with Expo (e.g., expo start, expo build)

### How to Test
- Run `npm install` to install updated dependencies
- Run `expo start` to launch the Expo development environment
- Test the app on both Android and iOS simulators/emulators or using the Expo Go app on a physical device
- Ensure that all current functionality and components work correctly in the Expo environment
